### PR TITLE
[releases] New, email automation for apache releases

### PR DESCRIPTION
**Pull request by dpgaspar**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [X] Documentation

### SUMMARY
Facilitate all the email process related with the Apache voting process.
Simple python script (with no dependencies) that generates and sends all the required emails during the voting process  

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by villebro** [comment]
Awesome! 🎉

File: RELEASING/email_templates/announce.tmpl [hunk 1]
```diff
@@ -0,0 +1,39 @@
+To: $receiver_email
```
Comment (mistercrunch) [inline RELEASING/email_templates/announce.tmpl]: Oh, interesting to see `string.Template` here. I'd have gone Jinja, but that's probably overkill, whatever works.

Comment (dpgaspar) [reply RELEASING/email_templates/announce.tmpl]: trying to keep it away from any requirements, let's see if I can pull it off. Already feeling the need for a loop block on the template to render a list of binding votes

Comment (dpgaspar) [reply RELEASING/email_templates/announce.tmpl]: ended up using Jinja2 and Click much cleaner and maintainable code now

**Comment by villebro** [comment]
This looks good, but let me digest this for a while. My initial feeling is that sending the emails from a script feels slightly dangerous, and could perhaps just be done by rendering the message to stdout and sent manually, but sending from CLI might OTOH might be a pretty good solution (less manual steps -> less risk for errors). Would be curious to hear other people's thoughts on this approach.

**Comment by dpgaspar** [comment]
@villebro I could easily add an extra argument to override the SMTP `To:`, that way we can first send to our private email and check if everything is ok, then rerun without the override.

btw I sent my last 2 emails regarding 0.35.1 using this

**Comment by dpgaspar** [comment]
Added an option to override the default receiver email, this way one can send a preview email to his personal mail box first

Note also that before sending any email the full SMTP message is displayed for the user to confirm

**Review by villebro — APPROVED** [review]
This looks really solid 👍  As it seems we usually need to put in an extra vote reminder, both PPMC and IPMC, perhaps we should have a template for that, too? However, that can be left for later.

**Comment by dpgaspar** [comment]
Thanks @villebro. True it would be a nice to have, yet there is the challenge to send the reminder email attached to the email thread

**Comment by villebro** [comment]
You're right @dpgaspar, didn't come to think of that. Perhaps we can live with writing manual reminders.. 😆

**Pull request merged**